### PR TITLE
added docs and completed typespec.

### PR DESCRIPTION
**Pull request by Fl4m3Ph03n1x**

### CODE REVIEW

**Review by FrancescoSaverioZuppichini — COMMENTED** [review]
Minor comments

File: darwin/importer/importer.py [hunk 1]
```diff
@@ -103,7 +105,26 @@ def import_annotations(
...
     append: bool,
-):
+) -> None:
+    """
+    Imports the given given Annotations into the given Dataset.
+
+    Parameters
+    ----------
+    dataset : RemoteDataset
+        Dataset where the Annotations will be imported to.
```
[the remark this answers is not in the document]
Comment (FrancescoSaverioZuppichini) [reply darwin/importer/importer.py:115]: The parameters are correct, but we are **missing type for dataset**

[the remark this answers is not in the document]
Comment (FrancescoSaverioZuppichini) [reply darwin/importer/importer.py:115]: Super!

**Review by FrancescoSaverioZuppichini — COMMENTED** [review]
All good!

File: darwin/importer/importer.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Fl4m3Ph03n1x) [reply darwin/importer/importer.py:115]: Fixed !

**Review by FrancescoSaverioZuppichini — APPROVED** [review]
Approved

**Pull request merged**